### ISSUE TITLE
[Remote-SSH Bug]: 'Open File' ' .. ' up directory behaviour - Unable to click back up the directory structure.

### ISSUE THREAD

**Issue by Darren-Chambers** [microsoft/vscode-remote-release#9908]
### Is there an existing issue for this bug?

- [X] I have searched the existing issues

### Required Troubleshooting Steps

- [X] I have followed these troubleshooting steps
- [ ] I have tried both values of the `remote.SSH.useLocalServer` setting

### Connect Locally

It connects successfully

### Expected Behavior

After updating to 1.89.1, using a remote connection I am having an unusual problem with 'Open File'.
I can successfully walk down the directory tree, however if I try to walk back up the tree using ' .. ' I can only come up one directory; subsequent clicks on the ' .. ' are unresponsive.
<img width="462" alt="vsc_up_directory" src="https://github.com/microsoft/vscode-remote-release/assets/74759908/aaf2f8ca-1fa6-452f-b049-967fc8cafa2a">

### Steps To Reproduce

1. Make a remote connection to a Linux host.
2. Click 'Open File'
3. Walk several directories down the directory tree.
4. Try to walk back up using the ' .. ' , first click should be successful, subsequent clicks unresponsive.

### Remote-SSH Log

<details>
<summary>Remote-SSH Log</summary>
<p>

```
[16:02:50.281] Log Level: 2
[16:02:50.294] SSH Resolver called for "ssh-remote+7b22686f73744e616d65223a225a7572696368227d", attempt 1
[16:02:50.295] "remote.SSH.useLocalServer": false
[16:02:50.295] "remote.SSH.useExecServer": false
[16:02:50.296] "remote.SSH.showLoginTerminal": false
[16:02:50.296] "remote.SSH.remotePlatform": REMOVED
[16:02:50.296] "remote.SSH.path": undefined
[16:02:50.296] "remote.SSH.configFile": undefined
[16:02:50.296] "remote.SSH.useFlock": true
[16:02:50.296] "remote.SSH.lockfilesInTmp": false
[16:02:50.296] "remote.SSH.localServerDownload": auto
[16:02:50.296] "remote.SSH.remoteServerListenOnSocket": false
[16:02:50.296] "remote.SSH.showLoginTerminal": false
[16:02:50.297] "remote.SSH.defaultExtensions": []
[16:02:50.299] "remote.SSH.loglevel": 2
[16:02:50.300] "remote.SSH.enableDynamicForwarding": true
[16:02:50.300] "remote.SSH.enableRemoteCommand": false
[16:02:50.300] "remote.SSH.serverPickPortsFromRange": {}
[16:02:50.300] "remote.SSH.serverInstallPath": {}
[16:02:50.303] VS Code version: 1.89.1
[16:02:50.304] Remote-SSH version: remote-ssh@0.109.0
[16:02:50.304] win32 x64
[16:02:50.310] SSH Resolver called for host: ******
[16:02:50.310] Setting up SSH remote "******"
[16:02:50.313] Using commit id "dc96b837cf6bb4af9cd736aa3af08cf8279f7685" and quality "stable" for server
[16:02:50.316] Install and start server if needed
[16:02:50.320] Checking ssh with "C:\Windows\system32\ssh.exe -V"
[16:02:50.322] Got error from ssh: spawn C:\Windows\system32\ssh.exe ENOENT
[16:02:50.322] Checking ssh with "C:\Windows\ssh.exe -V"
[16:02:50.323] Got error from ssh: spawn C:\Windows\ssh.exe ENOENT
[16:02:50.324] Checking ssh with "C:\Windows\System32\Wbem\ssh.exe -V"
[16:02:50.325] Got error from ssh: spawn C:\Windows\System32\Wbem\ssh.exe ENOENT
[16:02:50.325] Checking ssh with "C:\Windows\System32\WindowsPowerShell\v1.0\ssh.exe -V"
[16:02:50.326] Got error from ssh: spawn C:\Windows\System32\WindowsPowerShell\v1.0\ssh.exe ENOENT
[16:02:50.326] Checking ssh with "C:\Windows\System32\OpenSSH\ssh.exe -V"
[16:02:50.437] > OpenSSH_for_Windows_8.6p1, LibreSSL 3.4.3

[16:02:50.442] Running script with connection command: "C:\Windows\System32\OpenSSH\ssh.exe" -T -D 62601 "*****" bash
[16:02:50.444] Terminal shell path: C:\Windows\System32\cmd.exe
[16:02:51.574] > b88d8c96b72b: running
> ]0;C:\Windows\System32\cmd.exe
[16:02:51.574] Got some output, clearing connection timeout
[16:02:51.618] > Acquiring lock on /root/.vscode-server/bin/dc96b837cf6bb4af9cd736aa3af08cf8279f7685/vscode-remote-lock.root.dc96b837cf6bb4af9cd736aa3af08cf8279f7685
[16:02:51.633] > Found existing installation at /root/.vscode-server/bin/dc96b837cf6bb4af9cd736aa3af08cf8279f7685...
> Checking /root/.vscode-server/.dc96b837cf6bb4af9cd736aa3af08cf8279f7685.log and /root/.vscode-server/.dc96b837cf6bb4af9cd736aa3af08cf8279f7685.pid for a running server
> Looking for server with pid: 1110062
[16:02:51.711] > Found running server...
>  
> *
> * Visual Studio Code Server
> *
> * By using the software, you agree to
> * the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
> * the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
> *
>  
> Checking server status on port 44783 with wget
> b88d8c96b72b: start
> SSH_AUTH_SOCK====
> DISPLAY====
> webUiAccessToken====
> listeningOn==44783==
> osReleaseId==centos==
> arch==x86_64==
> vscodeArch==x64==
> bitness==64==
> tmpDir==/run/user/0==
> platform==linux==
[16:02:51.727] > 
> unpackResult====
> didLocalDownload==0==
> downloadTime====
> installTime====
> extInstallTime====
> serverStartTime====
> connectionToken==aaa1a11a-11a1-1aaa-aaaa-1111a1111a11==
> b88d8c96b72b: end
[16:02:51.728] Received install output: 
SSH_AUTH_SOCK====
DISPLAY====
webUiAccessToken====
listeningOn==44783==
osReleaseId==centos==
arch==x86_64==
vscodeArch==x64==
bitness==64==
tmpDir==/run/user/0==
platform==linux==
unpackResult====
didLocalDownload==0==
downloadTime====
installTime====
extInstallTime====
serverStartTime====
connectionToken==aaa1a11a-11a1-1aaa-aaaa-1111a1111a11==

[16:02:51.728] Remote server is listening on port 44783
[16:02:51.728] Parsed server configuration: {"serverConfiguration":{"remoteListeningOn":{"port":44783},"osReleaseId":"centos","arch":"x86_64","webUiAccessToken":"","sshAuthSock":"","display":"","tmpDir":"/run/user/0","platform":"linux","connectionToken":"aaa1a11a-11a1-1aaa-aaaa-1111a1111a11"},"installUnpackCode":""}
[16:02:51.738] Starting forwarding server. local port 62603 -> socksPort 62601 -> remotePort 44783
[16:02:51.740] Forwarding server listening on port 62603
[16:02:51.740] Waiting for ssh tunnel to be ready
[16:02:51.742] > 
[16:02:51.743] [Forwarding server port 62603] Got connection 0
[16:02:51.745] Tunneled port 44783 to local port 62603
[16:02:51.745] Resolved "ssh-remote+7b22686f73744e616d65223a225a7572696368227d" to "port 62603"
[16:02:51.759] ------




[16:02:51.776] [Forwarding server port 62603] Got connection 1
[16:02:51.872] [Forwarding server port 62603] Got connection 2
[16:02:52.833] [Forwarding server port 62603] Got connection 3

```

</p>
</details>

**Comment by sudoofus** [comment]
Looks like this is fixed in v1.90.2

**Comment by roblourens** [comment]
Thanks
